runtime/src/accounts_db.rs

```diff
@@ -7206,6 +7206,15 @@ impl AccountsDb {
         slots_per_epoch: Slot,
         mut stats: &mut crate::accounts_hash::HashStats,
     ) {
+        // Nothing to do if ancient append vecs are enabled.
+        // Ancient slots will be visited by the ancient append vec code and dealt with correctly.
```

Comment:
@jeffwashington: I would like to learn more about this code path.  Can you give me the code pointer that handles this?  Specifically want to know whether the new format need to handle this as well based on whether this is handled inside append-vec level or accounts file level.